code/modules/mob/living/simple_animal/friendly/drone/drones_as_items.dm

```diff
@@ -62,4 +62,6 @@
 		D.equip_to_slot_or_del(new_hat, SLOT_HEAD)
 	D.flags_1 |= (flags_1 & ADMIN_SPAWNED_1)
 	D.key = user.key
+	message_admins("[ADMIN_LOOKUPFLW(user)] has taken possession of a [src] in [AREACOORD(src)].")
```

Comment:
`a [src]` will produce "a the drone shell", either just `[src]` for "the drone shell" or `\a [src]` for "a drone shell"